test/integration/targets/azure_rm_sqlserver/aliases

```diff
@@ -1,3 +1,6 @@
 cloud/azure
 destructive
 shippable/azure/group4
+azure_rm_sqlserver_facts
+azure_rm_sqldatabase
```

Comment:
sqldatabase_facts?

Reply:
not yet merged, but we could add in advance?